### ISSUE TITLE
isort array options are specified as string

### ISSUE THREAD

**Issue by onepan** [ryantam626/jupyterlab_code_formatter#164]
**Checklist prior to opening an issue**
- [x] I have followed fully the installation steps laid out in the documentation site.
- [x] I have restarted jupyterlab.
- [x] I have read the FAQ section in the documentation site.

**Describe the bug**
The current schema for `isort` settings specifies that (e.g.) `known_first_party` is a string instead of an array. The value is passed to `isort.code` which expects a list of strings

**Diagnostic commands**
- `conda env export`

```yaml
name: test
channels:
  - conda-forge
  - defaults
dependencies:
  - _libgcc_mutex=0.1=conda_forge
  - _openmp_mutex=4.5=1_gnu
  - appdirs=1.4.3=py_1
  - argon2-cffi=20.1.0=py38h1e0a361_1
  - attrs=19.3.0=py_0
  - backcall=0.2.0=pyh9f0ad1d_0
  - backports=1.0=py_2
  - backports.functools_lru_cache=1.6.1=py_0
  - black=19.10b0=py_4
  - bleach=3.1.5=pyh9f0ad1d_0
  - brotlipy=0.7.0=py38h1e0a361_1000
  - ca-certificates=2020.6.20=hecda079_0
  - certifi=2020.6.20=py38h32f6830_0
  - cffi=1.14.1=py38h5bae8af_0
  - chardet=3.0.4=py38h32f6830_1006
  - click=7.1.2=pyh9f0ad1d_0
  - cryptography=3.0=py38h766eaa4_0
  - decorator=4.4.2=py_0
  - defusedxml=0.6.0=py_0
  - entrypoints=0.3=py38h32f6830_1001
  - icu=67.1=he1b5a44_0
  - idna=2.10=pyh9f0ad1d_0
  - importlib-metadata=1.7.0=py38h32f6830_0
  - importlib_metadata=1.7.0=0
  - ipykernel=5.3.4=py38h23f93f0_0
  - ipython=7.17.0=py38h1cdfbd6_0
  - ipython_genutils=0.2.0=py_1
  - isort=5.3.0=py38h32f6830_0
  - jedi=0.15.2=py38_0
  - jinja2=2.11.2=pyh9f0ad1d_0
  - json5=0.9.4=pyh9f0ad1d_0
  - jsonschema=3.2.0=py38h32f6830_1
  - jupyter_client=6.1.6=py_0
  - jupyter_core=4.6.3=py38h32f6830_1
  - jupyterlab=2.2.2=py_0
  - jupyterlab_code_formatter=1.3.5=py_0
  - jupyterlab_server=1.2.0=py_0
  - ld_impl_linux-64=2.34=hc38a660_9
  - libblas=3.8.0=17_openblas
  - libcblas=3.8.0=17_openblas
  - libffi=3.2.1=he1b5a44_1007
  - libgcc-ng=9.3.0=h24d8f2e_14
  - libgfortran-ng=7.5.0=hdf63c60_14
  - libgomp=9.3.0=h24d8f2e_14
  - liblapack=3.8.0=17_openblas
  - libopenblas=0.3.10=pthreads_hb3c22a3_4
  - libsodium=1.0.18=h516909a_0
  - libstdcxx-ng=9.3.0=hdf63c60_14
  - libuv=1.38.0=h516909a_0
  - markupsafe=1.1.1=py38h1e0a361_1
  - mistune=0.8.4=py38h1e0a361_1001
  - nbconvert=5.6.1=py38h32f6830_1
  - nbformat=5.0.7=py_0
  - ncurses=6.2=he1b5a44_1
  - nodejs=14.7.0=h568c755_0
  - notebook=6.1.1=py38h32f6830_0
  - numpy=1.19.1=py38h8854b6b_0
  - openssl=1.1.1g=h516909a_1
  - packaging=20.4=pyh9f0ad1d_0
  - pandoc=2.10.1=h516909a_0
  - pandocfilters=1.4.2=py_1
  - parso=0.8.0=pyh9f0ad1d_0
  - pathspec=0.8.0=pyh9f0ad1d_0
  - pexpect=4.8.0=py38h32f6830_1
  - pickleshare=0.7.5=py38h32f6830_1001
  - pip=20.2.1=py_0
  - prometheus_client=0.8.0=pyh9f0ad1d_0
  - prompt-toolkit=3.0.5=py_1
  - ptyprocess=0.6.0=py_1001
  - pycparser=2.20=pyh9f0ad1d_2
  - pygments=2.6.1=py_0
  - pyopenssl=19.1.0=py_1
  - pyparsing=2.4.7=pyh9f0ad1d_0
  - pyrsistent=0.16.0=py38h1e0a361_0
  - pysocks=1.7.1=py38h32f6830_1
  - python=3.8.5=h4d41432_2_cpython
  - python-dateutil=2.8.1=py_0
  - python_abi=3.8=1_cp38
  - pyzmq=19.0.2=py38ha71036d_0
  - readline=8.0=he28a2e2_2
  - regex=2020.7.14=py38h1e0a361_0
  - requests=2.24.0=pyh9f0ad1d_0
  - send2trash=1.5.0=py_0
  - setuptools=49.2.1=py38h32f6830_0
  - six=1.15.0=pyh9f0ad1d_0
  - sqlite=3.32.3=hcee41ef_1
  - terminado=0.8.3=py38h32f6830_1
  - testpath=0.4.4=py_0
  - tk=8.6.10=hed695b0_0
  - toml=0.10.1=pyh9f0ad1d_0
  - tornado=6.0.4=py38h1e0a361_1
  - traitlets=4.3.3=py38h32f6830_1
  - typed-ast=1.4.1=py38h516909a_0
  - urllib3=1.25.10=py_0
  - wcwidth=0.2.5=pyh9f0ad1d_1
  - webencodings=0.5.1=py_1
  - wheel=0.34.2=py_1
  - xz=5.2.5=h516909a_1
  - zeromq=4.3.2=he1b5a44_3
  - zipp=3.1.0=py_0
  - zlib=1.2.11=h516909a_1006
prefix: /home/testuser/.conda/envs/test
```

- `jupyter labextension list`
```
JupyterLab v2.2.2
Known labextensions:
   app dir: /home/testuser/.conda/envs/test/share/jupyter/lab
        @ryantam626/jupyterlab_code_formatter v1.3.5  enabled  OK
```

- `jupyter serverextension list`
```
config dir: /home/testuser/.jupyter
    jupyterlab_code_formatter  enabled 
    - Validating...
      jupyterlab_code_formatter  OK
config dir: /home/testuser/.conda/envs/test/etc/jupyter
    jupyterlab  enabled 
    - Validating...
      jupyterlab 2.2.2 OK
```

**Additional context**
Editing the schema file [settings.json](https://github.com/ryantam626/jupyterlab_code_formatter/blob/v1.3.5/labextension/schema/settings.json) and changing `isort.known_first_party` to 
```json
"known_first_party": {
    "type": "array",
    "element": "string"
}
```
solves the issue. All the option types are listed here https://github.com/timothycrosley/isort/wiki/isort-Settings

**Comment by ryantam626** [comment]
Hi, thanks for the bug report, you could have opened a pull request :wink:

**Comment by ryantam626** [comment]
Should be fixed in version 1.3.6 now

**Comment by onepan** [comment]
Thanks! Confirmed it's fixed. And yeah, sorry, I don't know why I didn't just do that.. next time!
